File: tooltip-action/src/main/java/com/github/kostasdrakonakis/listeners/TooltipAnimationListener.java
package com.github.kostasdrakonakis.listeners;

import android.animation.Animator;
import android.view.View;

public interface TooltipAnimationListener {
    void onAnimationEntered(View view, Animator.AnimatorListener animatorListener);

    void onAnimationExited(View view, Animator.AnimatorListener animatorListener);
}

File: tooltip-action/src/main/java/com/github/kostasdrakonakis/listeners/TooltipAnimationListener.kt
package com.github.kostasdrakonakis.listeners

import android.animation.Animator
import android.view.View

interface TooltipAnimationListener {
    fun onAnimationEntered(view: View, animatorListener: Animator.AnimatorListener)
    fun onAnimationExited(view: View, animatorListener: Animator.AnimatorListener)
}